src/openswathalgo/include/OpenMS/OPENSWATHALGO/OpenSwathAlgoConfig.h

```diff
@@ -0,0 +1,42 @@
+
+#ifndef OPENSWATHALGO_DLLAPI_H
```

Comment:
this file seems out of place here

Reply:
good catch. its a generated file and I accidently added it